src/state/actionProducers/showUpdateNotification.ts

```diff
@@ -0,0 +1,23 @@
+import PluginUpdateCheckerPlugin from 'src/main';
+import { store } from '..';
+import { ObsidianApp } from '../obsidianReducer';
+import { fetchReleases } from './fetchReleases';
+
+export const showUpdateNotificationMiddleware = () => (next: any) => (action: any) => {
+    const result = next(action);
+
+    if (action.type === fetchReleases.fulfilled.type) {
```

Comment:
By default this plugin checks for updates every 30 minutes which would trigger the notification again even if there's no new updates. Showing the notification once or if there's any new updates could solve the problem

If you want to fix this, then to help with testing create a `.env.local.dev` file in the root of this repository and add the line `OBSIDIAN_APP_POLLING_FREQUENCY_MULTIPLIER=30000`